src/librustc_mir/util/pretty.rs

```diff
@@ -467,6 +467,59 @@ fn write_scope_tree(
 ) -> io::Result<()> {
     let indent = depth * INDENT.len();
 
+    let parent_data = &mir.source_scopes[parent];
```

Comment:
This code is moved up so we also print locals in the root scope.

Reply:
We should try to get #56278 merged first, I think.

Reply:
That PR seems to do the same thing though, maybe I can just drop this commit?

Reply:
Oh, if that doesn't block anything else in this PR, sure!

Reply:
Done.